src/rgw/rgw_tracer.cc

```diff
@@ -0,0 +1,133 @@
+
+#include "rgw_tracer.h"
+
+using jConfig = jaegertracing::Config;
+using jLogger = jaegertracing::logging::Logger;
+
+namespace RGW_Tracing{
+
+    /*
+    //  Tracer
+    //
+    */
+    const std::shared_ptr<opentracing::Tracer> Tracer::noop_tracer = opentracing::MakeNoopTracer();
+    TracerConfiguration Tracer::tracer_conf;
+    
+    Tracer::Tracer(jConfig& conf):open_tracer(jaegertracing::Tracer::make(conf)){}
+    Tracer::Tracer(
+    std::string& service_name, 
+    const jConfig& conf, 
+    const std::shared_ptr<jLogger>& logger)
+    {
+        this->open_tracer = jaegertracing::Tracer::make(service_name, conf, logger);
```

Comment:
usually there is no need to prefix data members with "this->" (unless there could be some confusion with the names)

Reply:
also, you can move this to the init line, just line the first ctor

Reply:
done in 7482e24